internal/dataplane/configfetcher/kongrawstate_test.go

```diff
@@ -15,6 +16,12 @@ import (
 	"github.com/kong/kubernetes-ingress-controller/v3/internal/dataplane/kongstate"
 )
 
+func buildCustomEntityWithObject(entityType string, obj custom.Object) custom.Entity {
```

Comment:
Can we already accept the `entityType` as `custom.Type` instead of `string`? This will give the caller a hint what's expected instead of leaving it space for misuse.

WDYT?

Reply:
updated to use `custom.Type`.